src/main/archetype/ui.frontend/package.json

```diff
@@ -1,56 +1,60 @@
-    },
-    "devDependencies": {
-      "@babel/core": "^7.0.0",
-      "@babel/plugin-proposal-class-properties": "^7.3.3",
-      "@babel/plugin-proposal-object-rest-spread": "^7.3.2",
-      "acorn": "^6.1.0",
-      "aem-clientlib-generator": "^1.4.3",
-      "autoprefixer": "^9.2.1",
-      "browserslist": "^4.2.1",
-      "clean-webpack-plugin": "^3.0.0",
-      "copy-webpack-plugin": "^5.0.4",
-      "css-loader": "^3.0.0",
-      "cssnano": "^4.1.10",
-      "html-webpack-plugin": "^3.2.0",
-      "mini-css-extract-plugin": "^0.4.4",
-      "node-sass": "^4.11.0",
-      "optimize-css-assets-webpack-plugin": "^5.0.1",
-      "postcss-loader": "^3.0.0",
-      "sass": "^1.17.2",
-      "sass-loader": "^7.1.0",
-      "source-map-loader": "^0.2.4",
-      "style-loader": "^0.14.1",
-      "terser-webpack-plugin": "^1.4.1",
-      "ts-loader": "^5.3.3",
-      "tsconfig-paths-webpack-plugin": "^3.2.0",
-      "tslint": "^5.13.0",
-      "tslint-loader": "^3.5.4",
-      "tslint-webpack-plugin": "^2.0.2",
-      "typescript": "^3.3.3333",
-      "webpack": "^4.27.1",
-      "webpack-cli": "^3.1.2",
-      "webpack-dev-server": "^3.9.0",
-      "webpack-import-glob-loader": "^1.6.3",
-      "webpack-merge": "^4.2.1"
-    },
-    "dependencies": {},
-    "browserslist": [
-      "last 2 version",
-      "> 1%"
-    ]
-  }
+  "name": "aem-maven-archetype",
+  "version": "1.0.0",
+  "description": "Generates an AEM Frontend project with Webpack",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/aem-project-archetype"
+  },
+  "private": true,
+  "main": "src/main/webpack/site/main.ts",
+  "license": "SEE LICENSE IN LICENSE.txt",
+  "scripts": {
+    "dev": "webpack -d --env dev --config ./webpack.dev.js && clientlib --verbose",
+    "prod": "webpack -p --config ./webpack.prod.js && clientlib --verbose",
+    "start": "webpack-dev-server --open --config ./webpack.dev.js"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.3.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.3.2",
+    "@typescript-eslint/eslint-plugin": "^2.6.0",
+    "@typescript-eslint/parser": "^2.6.0",
+    "acorn": "^6.1.0",
+    "aem-clientlib-generator": "^1.4.3",
+    "autoprefixer": "^9.2.1",
+    "browserslist": "^4.2.1",
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^5.0.4",
+    "css-loader": "^3.0.0",
+    "cssnano": "^4.1.10",
+    "eslint": "^6.6.0",
+    "eslint-loader": "^3.0.2",
+    "html-webpack-plugin": "^3.2.0",
+    "mini-css-extract-plugin": "^0.4.4",
+    "node-sass": "^4.11.0",
+    "optimize-css-assets-webpack-plugin": "^5.0.1",
+    "postcss-loader": "^3.0.0",
+    "sass": "^1.17.2",
+    "sass-loader": "^7.1.0",
+    "source-map-loader": "^0.2.4",
+    "style-loader": "^0.14.1",
+    "terser-webpack-plugin": "^1.4.1",
+    "ts-loader": "^5.3.3",
+    "tsconfig-paths-webpack-plugin": "^3.2.0",
+    "tslint": "^5.13.0",
```

Comment:
@jbmatheson - can the tslint dev dependencies be removed?

Reply:
@richardhand - Good catch. I have now removed `tslint1` `tslint-loader` and `tslint-webpack-plugin` as they are no longer needed.